apps/api-documenter/src/yaml/OfficeYamlDocumenter.ts

```diff
@@ -148,4 +147,16 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
     }
     return text;
   }
+
+  private _generateSnippetText(snippets: string[]): string {
```

Comment:
>_generateSnippetText [](start = 10, length = 20)

This should perhaps be called `_generateExamplesSnippetText` because it generates a specific block of text.

Reply:
Makes sense. Changing.